index.html

```diff
@@ -20,6 +20,7 @@ <h1 id="quote">...</h1>
     <p class="credits"><small>A Six-Sided Space Ninja Production by <a href="http://spaceninja.com/">Scott Vandehey</a> and <a href="http://sixsided.org/">Miles Johnson</a>. You can follow us on Twitter: <a href="http://twitter.com/spaceninja">@spaceninja</a> and <a href="http://twitter.com/sixsided">@sixsided</a>.</small></p>
 </footer>
 
+<script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
```

Comment:
That's a great question. I'll file a second ticket. I seem to remember something like this for jqueryCDN, where you have a javascript thing that checks for the existance of a function from the loaded file, and if it doesn't' exist, then you load a local copy instead of the CDN.